### ISSUE TITLE
disableSecureView not working in iOS

### ISSUE THREAD

**Issue by taehyun-kim-tenuto** [killserver/react-native-screenshot-prevent#18]
Preventing screenshot is okay but once i disable screenshot, rollback is impossible
plus, example is not correct, there is no addEventListener in the module

`declare module 'react-native-screenshot-prevent' {
  export function enabled(value: boolean): void
  export function enableSecureView(): void;
  export function disableSecureView(): void;
}`

sampleCode

`useEffect(() => {
    if (isFocused) {
      RNPreventScreenshot.enableSecureView();
    } else {
      RNPreventScreenshot.disableSecureView();
      RNPreventScreenshot.enabled(false);
    }

    return () => {
      console.log('unmounted22');
      // RNPreventScreenshot.enabled(false);
      RNPreventScreenshot.disableSecureView();
    };
  }, [isFocused]);
`

**Comment by RicSilva83** [comment]
I changed this in RNScreenshotPrevent.m and it seems to work.


```
-(void) removeSecureTextFieldFromView:(UIView *) view {
    for(UITextField *subview in view.subviews){
        if([subview isMemberOfClass:[UITextField class]]){
            if(subview.secureTextEntry == TRUE){
                [subview removeFromSuperview];
                subview.secureTextEntry = FALSE;
                secureField.userInteractionEnabled = TRUE;
            }
        }
    }
}

#pragma mark - Public API

RCT_EXPORT_METHOD(enabled:(BOOL) _enable) {
    self->enabled = _enable;
}

/** adds secure textField view */
RCT_EXPORT_METHOD(enableSecureView){
    if(secureField.secureTextEntry == false){
        UIView *view = [UIApplication sharedApplication].keyWindow.rootViewController.view;
        for(UIView *subview in view.subviews){
            [self addSecureTextFieldToView:subview];
        }
    }
}

/** removes secure textField from the view */
RCT_EXPORT_METHOD(disableSecureView) {
    secureField.secureTextEntry = false;
    UIView *view = [UIApplication sharedApplication].keyWindow.rootViewController.view;
            for(UIView *subview in view.subviews){
                [self removeSecureTextFieldFromView:subview];
            }
}
```

**Comment by killserver** [comment]
> I changed this in RNScreenshotPrevent.m and it seems to work.
> 
> ```
> -(void) removeSecureTextFieldFromView:(UIView *) view {
>     for(UITextField *subview in view.subviews){
>         if([subview isMemberOfClass:[UITextField class]]){
>             if(subview.secureTextEntry == TRUE){
>                 [subview removeFromSuperview];
>                 subview.secureTextEntry = FALSE;
>                 secureField.userInteractionEnabled = TRUE;
>             }
>         }
>     }
> }
> 
> #pragma mark - Public API
> 
> RCT_EXPORT_METHOD(enabled:(BOOL) _enable) {
>     self->enabled = _enable;
> }
> 
> /** adds secure textField view */
> RCT_EXPORT_METHOD(enableSecureView){
>     if(secureField.secureTextEntry == false){
>         UIView *view = [UIApplication sharedApplication].keyWindow.rootViewController.view;
>         for(UIView *subview in view.subviews){
>             [self addSecureTextFieldToView:subview];
>         }
>     }
> }
> 
> /** removes secure textField from the view */
> RCT_EXPORT_METHOD(disableSecureView) {
>     secureField.secureTextEntry = false;
>     UIView *view = [UIApplication sharedApplication].keyWindow.rootViewController.view;
>             for(UIView *subview in view.subviews){
>                 [self removeSecureTextFieldFromView:subview];
>             }
> }
> ```

fix. I get your fix and apply in version 1.1.3
